spec/unit/installer/xcode/pods_project_generator/pod_target_installer_spec.rb

```diff
@@ -1,5 +1,37 @@
 require File.expand_path('../../../../../spec_helper', __FILE__)
 
+module Pod
+  class Installer
+    class Xcode
+      class PodsProjectGenerator
+        describe PodTargetInstaller do
+          describe 'In presence of Documentation Catalog' do
```

Comment:
can this fold within the other tests below?

Reply:
I can't. describe 'In General' spec is too long. rubocop will generate errors if I add anything to it.